pkg/limitador/k8s_objects.go

```diff
@@ -60,6 +60,11 @@ func Deployment(limitador *limitadorv1alpha1.Limitador, storageConfigSecret *v1.
 		replicas = int32(*limitador.Spec.Replicas)
 	}
 
+	var affinity *v1.Affinity
+	if limitador.Spec.Affinity != nil {
+		affinity = limitador.Spec.Affinity
+	}
+
```

Comment:
nit: is this check necessary ? 🤔  I think we can just directly assign to the spec value at the pod template `Affinity: limitador.Spec.Affinity,` since the value will be `nil` anyway from the spec if it is empty 🤔